docs/how-to-use-gvfs.md

```diff
@@ -72,20 +72,21 @@ the path mapping and convert automatically.
 | `fs.gravitino.enableCredentialVending`                | Whether to enable credential vending for the Gravitino Virtual File System.                                                                                                                                                                                                                                          | `false`                                                        | No                                  | 0.9.0-incubating |
 | `fs.gravitino.client.`                                | The configuration key prefix for the Gravitino client config.                                                                                                                                                                                                                                                        | (none)                                                         | No                                  | 1.0.0            |
 | `fs.gravitino.filesetMetadataCache.enable`            | Whether to cache the fileset or fileset catalog metadata in the Gravitino Virtual File System. Note that this cache causes a side effect: if you modify the fileset or fileset catalog metadata, the client can not see the latest changes.                                                                          | `false`                                                        | No                                  | 1.0.0            |
+| `fs.gravitino.autoCreateLocation`                     | The configuration key for whether to enable auto-creation of fileset location when the server-side filesystem ops are disabled and the location does not exist.                                                                                                                                                      | `true`                                                         | No                                  | 1.0.1           |
```

Comment:
The `since version` is `1.1.0` because this PR will not be cherry-picked into branch-1.0.